app/controllers/api/v1/rents_controller.rb

```diff
@@ -7,6 +7,7 @@ def index
 
       def create
         rent = @current_user.rents.create!(rent_params)
+        RentMailer.new_rent_send(rent.id).deliver_later if rent
```

Comment:
this is not bad, but this uses active job (an integrated way on Rails to make things happened in background); we need to use sidekiq to do this, please [sidkie](https://github.com/mperham/sidekiq/wiki/Getting-Started)

Reply:
Internally, active job uses sidekiq also, you can do it  with a worker or this way, we will be uses a worker in the old way in a incoming task